server/tokenRegistryProxy.js

```diff
@@ -1,29 +1,95 @@
 require('isomorphic-fetch')
 const Sentry = require('@sentry/node')
 const {backendConfig} = require('./helpers/loadConfig')
-const cacheResults = require('./helpers/cacheResults')
+const chunk = require('./helpers/chunk')
+const Cache = require('./helpers/cache')
 
-const CACHE_TIMEOUT = 24 * 60 * 60 * 1000 // 1 day
+const REQUEST_CHUNK_SIZE = 100
```

Comment:
SUBJECTS_PER_REQUEST would imho be a more descriptive name

Reply:
Refactored